src/Form/FormAutosuggest.jsx

```diff
@@ -124,13 +127,12 @@ function FormAutosuggest({
   };
 
   const keyDownHandler = e => {
-    if (e.key === 'Escape') {
+    if (e.key === 'Escape' && isActive) {
       e.preventDefault();
 
       setState(prevState => ({
         ...prevState,
         dropDownItems: [],
-        errorMessage: !state.displayValue ? errorMessageText : '',
```

Comment:
[clarification] If removing the `errorMessage` state here and below, is it correct that this component will no longer be using the `errorMessageText` prop anymore?

Reply:
it seems to still be working on https://deploy-preview-2517--paragon-openedx.netlify.app/components/form/form-autosuggest/

![image](https://github.com/openedx/paragon/assets/112954497/18d52f4b-9f77-4492-af50-c1ade37ee8d4)

Reply:
We're not changing the errorMessage state in the keyDownHandler since the user will still be focused on the component even when they type esc (because this only closes the drop down) #2516. Therefore, we wouldn't want there to be any reported errors until they have left the component. 

We still are keeping errorMessageText prop since we use it in handleDocumentClick function above.